ezyrb/ann.py

```diff
@@ -143,7 +142,7 @@ def fit(self, points, values):
                 elif isinstance(criteria, float):  # stop criteria is float
                     if loss.item() < criteria:
                         flag = False
-
+            print(loss.item())
```

Comment:
Can we have something more customizable? Like decide the print frequency (`frequency_print` should be an optional argument in the constructor) like in this snippet
```
if epoch % frequency_print == 0 or epoch == 1:                                                                                
    print('[epoch {:05d}] {:.6e}'.format(self.trained_epoch, loss.item()))
```